components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -45,7 +45,8 @@ object ErrorPages {
  */
 enum class ErrorType(
     @StringRes val titleRes: Int,
-    @StringRes val messageRes: Int
+    @StringRes val messageRes: Int,
+    @StringRes val refreshButtonRes: Int = R.string.mozac_browser_errorpages_page_refresh
```

Comment:
nit: I see that we are adding this (`refreshButtonRes`) property but we are not using it, is there any reason?

Reply:
Whoops!  Fixed.